Core/ResponseHandler.php

```diff
@@ -55,7 +56,21 @@ public static function onSuccessResponse($oResponse, $oBackendService, $oOrder)
 
         if (!self::_isPostProcessingAction($oResponse)) {
             self::_updateOrder($oOrder, $oResponse, $oBackendService);
-            $oOrder->sendOrderByEmail();
+
+            try {
+                $oOrder->sendOrderByEmail();
+            } catch (SystemComponentException $oException) {
+                // this error occurs when the 'Azure' theme is activated and a non-3DS credit card transaction is made
+                // everything was actually successful, but the 'getThumbnailUrl' method does not exist on the BasketItem
+                // in the 'order_cust.tpl' file. There is no need to do anything in this case but normally continue with
+                // the response handling.
+                // Otherwise the exception is re-thrown.
+                $sAzureErrorMsg = "Function 'getThumbnailUrl' does not exist or is not accessible!";
+
+                if (strpos($oException->getMessage(), $sAzureErrorMsg) !== 0) {
```

Comment:
This is too strict to be robust enough. What if the message is modified and this falsely throws the exception?

Reply:
👍